### ISSUE TITLE
Missing docfields and descriptions

### ISSUE THREAD

**Issue by muniter** [frappe/frappe_docs#33]
The docs are lacking the list of docfields available. Something like what [ERPNext Docs have](https://docs.erpnext.com/docs/user/manual/en/customize-erpnext/articles/field-types) but maybe in the style of [FrappJS Fields](https://frappe.io/frappejs/docs/models/fields.md)

**Comment by gwhitney** [comment]
Indeed, I just noticed that the "Fieldtypes" link at the bottom of the https://frappeframework.com/docs/user/en/understanding-doctypes#docfield section is a dead link, and so was going to file an issue, when I saw this, which is clearly focused on the same thing.

As I said in #16, I would like to contribute a section to the documentation on using DocFields to tailor how users see and interact with Documents, and so I would be glad to submit a PR to fix up this dead link and alleviate this underlying issue, among other things.

However, it is not immediately clear what ideally to link to. As far as I can see, there is nothing in the ERPNext Fields documentation that doesn't actually apply directly to Frappe. So should I do a double PR that moves the field list from the ERPNext documentation into the Frappe Framework documentation, and references the Frappe Framework documentation from ERPNext? If so, what is the recommended way to make cross-references between the two documentation packages like this?

Or conversely, should I just reference the existing ERPNext documentation page the OP lists from the Frappe Framework docs? Again, if so, please provide some guidance on how such cross-references should be made in the Frappe Framework docs.

Finally, the OPs other link to FrappeJS docs raises other issues. I am unclear on the relationship between the Frappe Framework and FrappeJS. I see that FrappeJS has integrated documentation, while Frappe Framework has a separate documentation repository. But what is the status of shared topics? Is the fields list from FrappeJS documentation just reflecting an outdated shorter list of field types, and does it actually have all Frappe Framework fields? Or is FrappeJS really separate with its own separate list of fields that needs to be documented independently? Or should the field list documentation of Frappe Framework and FrappeJS be merged?

  Thanks in advance for providing some guidance as to where the definitive implementation of the field types in Frappe Framework, ERPNext (does ERPNext add any field types that are not in Frappe Framework itself), and FrappeJS resides, and where that field list (or lists, if there are separate ones) should be documented and I will be happy to submit a PR arranging the documentation (and making that currently dead link active) as you outline.

**Comment by muniter** [comment]
Hello, I wish I could be more of help, what I know:

1. ERPNext does not implemente additional field types.
1. FrappeJS is a **different** framework but very similar.

And as you've stated before, I wouldn't now how to contribute it, where to put it, etc. Hoping from some of ERPNext employees could shade some light into it.

**Comment by gwhitney** [comment]
Great, so I will not worry about FrappeJS at all, presuming it has independent implementation and documentation. If someone involved directly with one project or the other could tell us whether they would like the documentation of the list of field types to be in the Frappe documentation with a link from ERPNext, or in the ERPNext documentation with a link from Frappe, and what the format of such links between separate documentation repositories will be, I will be happy to issue an appropriate pull request. Thanks!
